### PR TITLE
fix APCu installation for the nightly build job

### PR DESCRIPTION
**Pull request by xabbuh**
| Q             | A
| ------------- | ---
| Branch?       | 4.4
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       | 
| License       | MIT
| Doc PR        |

### CODE REVIEW

**Comment by xabbuh** [comment]
follows #38610

**Review by derrabus — APPROVED** [review]
Just a CS issue.

File: .travis.yml [hunk 1]
```diff
@@ -157,15 +157,14 @@ before_install:
...
-              tfold ext.apcu install_apcu_dev 9c36db45100d4d27ec33072f4be90f1f5a0108b7 $INI
           else
-              tfold ext.apcu tpecl apcu-5.1.19 apcu.so $INI
               tfold ext.mongodb tpecl mongodb-1.6.16 mongodb.so $INI
               tfold ext.zookeeper tpecl zookeeper-0.7.2 zookeeper.so $INI
               tfold ext.amqp tpecl amqp-1.10.2 amqp.so $INI
               tfold ext.redis tpecl redis-5.2.2 redis.so $INI "no"
           fi
 
+              tfold ext.apcu tpecl apcu-5.1.19 apcu.so $INI
```
Comment (derrabus) [inline .travis.yml]: The indentation does not look right. 🧐

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @xabbuh.

**Pull request merged**